src/auto/task.rs

```diff
@@ -0,0 +1,333 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+#[cfg(any(feature = "v2_36", feature = "dox"))]
+use Cancellable;
+#[cfg(any(feature = "v2_36", feature = "dox"))]
+use Error;
+use ffi;
+use glib;
+#[cfg(any(feature = "v2_44", feature = "dox"))]
+use glib::object::Downcast;
+use glib::object::IsA;
+#[cfg(any(feature = "v2_44", feature = "dox"))]
+use glib::signal::SignalHandlerId;
+#[cfg(any(feature = "v2_44", feature = "dox"))]
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+#[cfg(any(feature = "v2_44", feature = "dox"))]
+use std::boxed::Box as Box_;
+use std::mem;
+#[cfg(any(feature = "v2_44", feature = "dox"))]
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct Task(Object<ffi::GTask, ffi::GTaskClass>);
```

Comment:
This is not really that useful, it needs deeper integration than what GIR could possibly autogenerate. It probably also makes most sense with some kind of futures integration (because that's exactly what a `GTask` is).